mobile/src/main/AndroidManifest.xml

```diff
@@ -87,6 +87,23 @@
             android:exported="false" />
         <receiver android:name=".shared.notifications.CancelNotificationBroadcastReceiver"
             android:exported="false" />
+
+        <provider
+            android:name="androidx.startup.InitializationProvider"
+            android:authorities="${applicationId}.androidx-startup"
+            android:exported="false"
+            tools:node="merge">
+            <meta-data
+                android:name="com.google.samples.apps.iosched.util.initializers.AndroidThreeTenInitializer"
+                android:value="androidx.startup" />
+            <meta-data
+                android:name="com.google.samples.apps.iosched.util.initializers.StrictModeInitializer"
+                android:value="androidx.startup" />
```

Comment:
These two used to be called before `onCreate` so that I didn't init these lazily as these will be initialized automatically before `onCreate` is called.